README.md

```diff
@@ -10,7 +10,11 @@ You need to have installed **docker** on your machine.
 
 To build the docker images locally, just launch:
 
-`mvn clean package`
+`$> mvn clean package`
```

Comment:
We don't plan to keep maven here, but it can be removed in the separate PR.

Reply:
Done